lib/react.dart

```diff
@@ -142,9 +142,9 @@ abstract class Component {
     setState({}, callback);
   }
 
-  /// Set [_nextState] to provided [newState] value and force a re-render.
+  /// Performs a shallow merge of [newState] into [_nextState], and triggers a re-render.
```

Comment:
This is a bit unclear - what's being merged is `newState` and `state` - setting `_nextState` is the mechanism for making that happen.